oap-server/server-query-plugin/promql-plugin/src/main/java/org/apache/skywalking/oap/query/promql/rt/PromQLExprQueryVisitor.java

```diff
@@ -440,6 +440,32 @@ private Entity buildEntity(Layer layer,
                 checkLabels(labelMap, LabelName.ENDPOINT);
                 entity.setEndpointName(labelMap.get(LabelName.ENDPOINT));
                 break;
+            case ServiceRelation:
+                checkLabels(labelMap, LabelName.DEST_SERVICE);
+                checkLabels(labelMap, LabelName.DEST_LAYER);
```

Comment:
```suggestion
                checkLabels(labelMap, LabelName.DEST_SERVICE, LabelName.DEST_LAYER);
```
We can check the labels at the same time.